vbscripts/popUp.js

```diff
@@ -0,0 +1,207 @@
+const project = [
+  {
+    projectTitle: 'Profesional Art Printing Data',
+    projectDiscription:
+      ' A daily selection of privately personalized reads; no accounts or sign-ups required. has'
```

Comment:
- [ optional] since the previous reviewer didnt mention this, i advice you to look into it. kindly try as much as possible for the desktop version to look like the figma design
![Screenshot (268)_LI](https://user-images.githubusercontent.com/65298048/180244229-c8c89128-a004-497a-a26f-22355948577a.jpg)